packages/base-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add `registerMethodActionHandlers` method to `Messenger`, and `RestrictedMessenger` for simplified bulk action handler registration ([#5927](https://github.com/MetaMask/core/pull/5927))
+  - Allows registering multiple action handlers at once by passing an array of method names
+  - Automatically binds methods to the controller instance
```

Comment:
Maybe we should clarify that the action handlers map to methods and say that this works for any messenger client? 

``` suggestion
  - Allows registering action handlers that map to methods on a messenger client at once by passing an array of method names
  - Automatically binds action handlers to the given messenger client
```